packages/backend/src/database/migrations/20250122161125_add_spotlight_visibility_column_to_catalog_search.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from 'knex';
+
+const CatalogTableName = 'catalog_search';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable(CatalogTableName, (table) => {
+        table.boolean('spotlight_show').defaultTo(true).notNullable();
+    });
+}
+
+export async function down(knex: Knex): Promise<void> {
+    await knex.schema.alterTable(CatalogTableName, (table) => {
```

Comment:
we normally check if column exists first, just to avoid unnecesary errors 

eg:
```
  if (await knex.schema.hasColumn(dashboards, 'slug')) {
        await knex.schema.alterTable(dashboards, (table) => {
            table.dropColumn('slug');
        });
    }
    ```

Reply:
TODO: Add this

Reply:
oops same commit as above but included this change request https://github.com/lightdash/lightdash/commit/1dbdbde9b4e4f416cdcdaeea2726f79ae43a15b5